collaborative/IOU/heat_index.ipynb

```diff
@@ -480,71 +756,115 @@
+   "id": "41d76861-93b3-41f7-ba3c-22c772813745",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "sim_med = (num_heatidx_projdays.median(dim='simulation').assign_coords({\"simulation\": \"simulation median\"}).expand_dims(\"simulation\"))\n",
-    "proj_trend = trendline_median(sim_med)"
+    "# visualize results\n",
+    "num_heatidx_projdays.hvplot.line(x='year', by='simulation', title='') *\\\n",
+    "trendline(num_heatidx_projdays_stats, kind='median').hvplot.line(x='year', color='black', line_dash='dashed', label='trendline')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9519ea9-54ec-4023-89a8-057e5c10cfca",
+   "metadata": {},
+   "source": [
+    "### 4c) Calculate the number of Heat Index days per month\n",
+    "Alternatively, let's determine the number of days per month with Heat Index values above the threshold, as this may be useful for seasonal information. We'll be using the daily maximum Heat Index data."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "41d76861-93b3-41f7-ba3c-22c772813745",
+   "id": "698f025f-6d81-43a8-9bd5-98dbda9e65a3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "num_heatidx_histmonths = (heatidx_hist_day >= hi_threshold).resample(time='1M').sum()\n",
+    "num_heatidx_histmonths.name = 'Days per month above Heat Index threshold of {}°F'.format(hi_threshold)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d0b730bc-cbd8-45b8-9e59-ea1575298550",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "num_heatidx_projdays.hvplot.line(x='year', by='simulation') *\\\n",
-    "proj_trend.hvplot.line(x='year', color='black', line_dash='dashed', label='trendline')"
+    "num_heatidx_histmonths.hvplot.line(x='time', by='simulation')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "83433596-3d67-4a68-8fb8-909e6e74e363",
+   "id": "3889ae2b-557f-46a4-853a-06ee96901981",
    "metadata": {},
    "source": [
-    "## Step 3: Export\n",
-    "Export any variable of interest here for your needs. Here we've selected the daily historical Heat Index values, but you can pass any variable of interest to `var_to_save` here to customize the data to export. "
+    "### 4d) Export summer counts of days per year\n",
+    "\n",
+    "Like steps 2c and 3c, we'll calculate the min, max, mean, and median trends and format a table for easy export. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "09b0a46e-0702-4657-9159-c1e7d0350dd2",
+   "id": "a3e08238-91cb-403b-b23f-ed2df3dd3060",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "var_to_save = heatidx_hist_day\n",
-    "df_export = var_to_save.drop('Lambert_Conformal').to_dataframe()\n",
-    "df_export"
+    "# calculate summary stats\n",
+    "num_heatidx_histdays_stats = compute_multimodel_stats(num_heatidx_histdays_stats)\n",
```

Comment:
This line throws an error later when trying to create the summary table because it makes multimodel stats on a table that already has multimodel stats, so the data (and importantly index) are repeated. I think you run

```
# calculate summary statistics
num_heatidx_histdays_stats = compute_multimodel_stats(num_heatidx_histdays)
```
in 4b, so you could re-run that in this cell.

Reply:
Confirming the error and that rerunning the lines Calvin suggested works.

Reply:
I actually modified this section of code to generate the table on the monthly data, but fixed the approach!